project.clj

```diff
@@ -10,16 +10,21 @@
 
   :description "SSL certificate management on the JVM."
 
+  :min-lein-version "2.7.1"
+
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.5"]
+                   :inherit [:managed-dependencies]}
+
   ;; Abort when version ranges or version conflicts are detected in
   ;; dependencies. Also supports :warn to simply emit warnings.
-  ;; requires lein 2.2.0+.
   :pedantic? :abort
 
-  :dependencies [[org.clojure/tools.logging "0.2.6" :exclusions [org.clojure/clojure]]
+  :dependencies [[org.clojure/tools.logging]
                  [org.bouncycastle/bcpkix-jdk15on "1.55"]
-                 [commons-codec "1.9"]
-                 [clj-time "0.7.0"]
```

Comment:
Isn't is still expected that if a codebase has a dependency on a particular version, that it specifies that version (or version range)?